herbert_core/admin/@opt_config_manager/opt_config_manager.m

```diff
@@ -159,7 +183,7 @@ function save_configurations(obj,varargin)
             % and return pc type.
             %
             % A pc type is a string, describing the computer from point of
-            % view of using it for high performance communications.
+            % view of using it for Horacing.
```

Comment:
`Horacing`?

Reply:
This term was used unofficially during Horace training courses. It sounds fisy so I think, its fun to have it in docs :)